LICENSES.third-party

```diff
@@ -29,37 +29,12 @@ TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
 SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 
-ddt
```

Comment:
Think this is removed. It used to be in `numba/testing/ddt.py` according to https://github.com/numba/numba/search?q=ddt

Reply:
yes, done in https://github.com/numba/numba/pull/5014